src/mmw/js/src/core/models.js

```diff
@@ -6,6 +6,20 @@ var Backbone = require('../../shim/backbone'),
     turfArea = require('turf-area'),
     utils = require('./utils');
 
+var handleNullInCallback = function(callbackFn) {
+    return _.wrap(callbackFn, function(fn, args) {
+        try {
+            fn(args);
+        } catch (e) {
+            if (e instanceof TypeError) {
+                console.log('Caught TypeError exception that implies a callback encountered a null reference --> ' + e.message);
```

Comment:
I think that this null reference problem *did* occur before, but it was much more frequent after I fixed the connected bug, so it seemed appropriate to address it. The most noticeable effect prior to this fix was that the console showed errors in place of this message, and I recall there was a minor UI problem as well.